templates/plugin/pcie_errors.conf.epp

```diff
@@ -0,0 +1,14 @@
+<Plugin pcie_errors>
```

Comment:
please add a comment like `# THIS FILE IS MANAGED BY PUPPET`

Reply:
"# Generated by Puppet" line is already generated by Puppet. Test case for default values ensures that this line is present.

Reply:
ah right, stupid me!